debug.log

```diff
@@ -0,0 +1,2 @@
+[1012/220316.564:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
we don't need to send the debug to be a part of the commit, this need to be taken out.